src/components/WalletStatementModal/index.js

```diff
@@ -13,21 +13,16 @@ import ROUTES from '../../ROUTES';
 import Navigation from '../../libs/Navigation/Navigation';
 import * as Report from '../../libs/actions/Report';
 
-class WalletStatementModal extends React.Component {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            isLoading: true,
-        };
-    }
+function WalletStatementModal({statementPageURL, session}) {
+    const [isLoading, setIsLoading] = useState(true);
+    const authToken = lodashGet(session, 'authToken', null);
 
     /**
      * Handles in-app navigation for iframe links
      *
      * @param {MessageEvent} e
      */
-    navigate(e) {
+    const navigate = (e) => {
```

Comment:
I know it was `e` before, but I would recommend to use more meaningful names for variables/parameters, so here something like `event` will be better. WDYT?